lib/fastlane/actions/xcodebuild.rb

```diff
@@ -112,8 +120,13 @@ def self.run(params)
           if hash_args = hash_to_args(params)
             xcodebuild_args += hash_args
           end
+
+          buildlog_path ||= params[:buildlog_path]
         end
 
+        # By default we put xcodebuild.log in the Logs folder
+        buildlog_path ||= File.expand_path("~/Library/Logs/fastlane/xcbuild/#{Time.now.strftime('%F')}/#{Process.pid}")
```

Comment:
Do not use `Time.now.strftime` without zone. Use one of `Time.zone.now.strftime`, `Time.now.strftime.current`, `Time.now.strftime.in_time_zone`, `Time.now.strftime.utc`, `Time.now.strftime.getlocal`, `Time.now.strftime.iso8601`, `Time.now.strftime.jisx0301`, `Time.now.strftime.rfc3339`, `Time.now.strftime.to_i`, `Time.now.strftime.to_f` instead.